integration/spark/spark2/build.gradle

```diff
@@ -86,35 +101,29 @@ def commonTestConfiguration = {
     classpath = project.sourceSets.test.runtimeClasspath
 }
 
-
 test {
+    dependsOn shadowJar
     configure commonTestConfiguration
     useJUnitPlatform {
         excludeTags 'integration-test'
     }
 }
 
-task integrationTest(type: Test) {
```

Comment:
Are we sure that same behavior is preserved?

Reply:
intergration tests are only present in `app` project. I've checked that.